### ISSUE TITLE
Migrate adobeio/adobeio-documentation to AdobeDocs

### ISSUE THREAD

**Issue by macdonst** [adobe/Adobe-Developer-Site#54]
The https://github.com/adobeio/adobeio-documentation repository contains a number of folders. Each folder corresponds to a product's documentation that is displayed on adobe.io. We need to migrate this mono-repo into multiple repos under the AdobeDocs organization.

In order to accomplish this we will need to complete a number of tasks:

1. Create a repo for each folder in adobeio/adobeio-documentation in AdobeDocs org.
1. Modify each manifest.json file in the new repo's to reflect the correct paths.
1. Update Kirby so it points to these new repos for displaying documentation.
1. Engage with the product teams in order to let them know that the docs location has changed and setup access for them to update their own documentation.

**Comment by MattLusher** [comment]
Thanks for the list, Simon! I've got a meeting scheduled with Simon Wex tomorrow, because they're working out details on how the new Helix setup is going to work (it's the replacement for Kirby). I'll annotate this if need be with any additional steps we might need.

**Comment by macdonst** [comment]
@MattLusher I'm sitting next to @simonwex right now and he's forwarded me the meeting invite. I think we can move forward with the splitting up of the repo before switching over to Helix. It'll probably be less risky to move in baby steps. Let's table that decision until tomorrow.

**Comment by MattLusher** [comment]
@macdonst Ah, the advantage of locality! Good to know we can proceed with breaking up the repo. My next concerns are about matters related to managing the repo: specifically, who do we make collaborators and who should admins, and if there are any specific GitHub policies or configurations we should set. For instance, I made the stage branch the default on adobeio-documentation so that people who weren't paying attention to their branches and simply made and committed changes wouldn't accidentally push them live in master; I also set a policy that all PRs to master needed a review.

**Comment by macdonst** [comment]
@mattlusher yeah, we should talk about that stuff as well and make sure it gets included in https://github.com/AdobeDocs/docs-guidelines.

**Comment by filmaj** [comment]
@macdonst @simonwex @MattLusher where are we at with this issue?
